Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM php:7.4-apache
+
+RUN set -ex; \
+	apt-get update && apt-get install -y \
+		libmcrypt-dev libxml2-dev libzip-dev libbz2-dev \
+	    zlib1g-dev libpng-dev libjpeg-dev \
+		zip unzip git mariadb-client nano;
+
+RUN set -ex; echo "memory_limit = 1024M\n" >> /usr/local/etc/php/php.ini
+RUN set -ex; echo "max_execution_time=3600" >> /usr/local/etc/php/php.ini
+
+RUN docker-php-ext-install gd intl pdo_mysql zip
+RUN docker-php-ext-enable gd intl pdo_mysql zip
+
+RUN a2enmod rewrite
+
+COPY --from=composer:2 /usr/bin/composer /usr/bin/composer
+
+COPY --chown=www-data:www-data . /var/www/html
+COPY dev-ops/docker/000-default.conf /etc/apache2/sites-available/000-default.conf
+COPY dev-ops/docker/entrypoint.sh /var/www/html/entrypoint.sh
+
+RUN runuser -u www-data -- make --directory=/var/www/html shopware-platform-files
+
+RUN pecl install xdebug-2.9.8;\
+    docker-php-ext-enable xdebug
```

Comment:
We are thinking about a base image to speed up the build process. Do you think, it would be a good idea to move this RUN a bit higher? I suggest to put this right behind the COPY for composer. Then all the steps up to this can be extracted into a new image.

Reply:
Due to the fact that the Playground project is not made for productive use, I think it's OK using Xdebug for the base image. Then Xdebug can also be installed in the layer directly after the COPY for Composer.

Reply:
I updated the docker container reference to the versioned container matching the release of 0.6